upgrade/run_upgrade_test.sh

```diff
@@ -30,32 +30,49 @@ ROOT=$(dirname "$WD")
 # Set up inputs needed by /istio/istio/tests/upgrade/test_crossgrade.sh
 # These environment variables are passed by /istio/test-infra/prow/cluster/jobs istio periodic upgrade jobs
 export SOURCE_HUB=${SOURCE_HUB:-"docker.io/istio"}
-export SOURCE_TAG=${SOURCE_TAG:-"1.3.0"}
-export SOURCE_RELEASE_PATH=${SOURCE_RELEASE_PATH:-"https://storage.googleapis.com/istio-build/dev"}
 export TARGET_HUB=${TARGET_HUB:-"docker.io/istio"}
+export SOURCE_TAG=${SOURCE_TAG:-"1.3.0"}
 export TARGET_TAG=${TARGET_TAG:-"1.3.4"}
+export SOURCE_RELEASE_PATH=${SOURCE_RELEASE_PATH:-"https://storage.googleapis.com/istio-build/dev"}
 export TARGET_RELEASE_PATH=${TAGET_RELEASE_PATH:-"https://storage.googleapis.com/istio-build/dev"}
 export INSTALL_OPTIONS=${INSTALL_OPTIONS:-"helm"}
 export FROM_PATH=${FROM_PATH:-"$(mktemp -d from_dir.XXXXXX)"}
 export TO_PATH=${TO_PATH:-"$(mktemp -d to_dir.XXXXXX)"}
 
-# Download and unpack istio release artifacts.
-function download_untar_istio_release() {
-  local url_path=${1}
-  local tag=${2}
-  local dir=${3:-.}
+function get_git_sha() {
+  local tag=${1}
+  GIT_SHA=""
 
   if [[ "${tag}" =~ "latest" ]];then
     release_version=$(echo "${tag}" | cut -d'_' -f1)
     GIT_SHA=$(curl "${url_path}/${release_version}-dev")
```

Comment:
`url_path` is not defined. I don't think `get_git_sha` function will actually run.

Reply:
Oops, forgot to pass that url_path

Reply:
What is an example `source_tag` where this code is executed?

Reply:
Our upgrade job is triggered here: https://github.com/istio/test-infra/blob/master/prow/cluster/jobs/all-periodics.yaml#L148

Reply:
an example of SOURCE_TAG: 1.4.4 or SOURCE_TAG: 1.5_latest